packages/fast-tooling-react/src/viewer/viewer-content/viewer-content.props.ts

```diff
@@ -14,7 +14,7 @@ export interface ViewerContentHandledProps extends ViewerContentManagedClasses {
     /**
      * The plugins available for manipulating the data mapping function
      */
-    plugins?: Array<Plugin<PluginProps>>;
```

Comment:
This is going to be somewhat of a hassle for rebasing the tooling v2 releases branch, if it's not critical can we hold on it until next week?

Reply:
I will break out this bit into a new PR and we can rebase post v2.  Besides, I feel you may want to test this a little more thoroughly than myself.